### ISSUE TITLE
The Rain Collector doesn't work

### ISSUE THREAD

**Issue by LLEVC** [Charles445/SimpleDifficulty#23]
**The Rain Collector block didn't work, although I used Weathers, Storms, and Tornadoes mod** so I think you should **add support for that mod.**

**Comment by vizthex123** [comment]
Agreed, I noticed it doesn't work with Weather2. Kind of odd it doesn't have mod compat...

**Comment by Charles445** [comment]
Weather2's API suggests that it's a laggy operation to check blocks for their weather, so it could be very tricky to implement without becoming a lag source
If I can figure out a way to safely implement it, I will

**Comment by vizthex123** [comment]
perhaps you could make it just detect the current weather, and fill up if it's raining/thunder/etc?

**Comment by LLEVC** [comment]
> Weather2's API suggests that it's a laggy operation to check blocks for their weather, so it could be very tricky to implement without becoming a lag source
> If I can figure out a way to safely implement it, I will

thank you ^-^
